eZ/Publish/SPI/Persistence/Content/Type/DeleteByParamsStruct.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\SPI\Persistence\Content\Type;
+
+use eZ\Publish\SPI\Persistence\ValueObject;
+
+class DeleteByParamsStruct extends ValueObject
+{
+    /**
+     * @var int
+     */
+    public $modifierId;
+
+    /**
+     * @var int
+     */
+    public $version;
```

Comment:
Is this property a versionId as suggest `@var int` in docblock? or object as suggest variable name? Or CT status?

Reply:
on second thoughts I think status would be better name